.github/workflows/test_url.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+for i in range(0,6):
```

Comment:
What happens if the domain is not up when i = 6? we need to throw an error

Reply:
Corrected in commit [6eb89bc](https://github.com/opensearch-project/opensearch-api-specification/pull/35/commits/6eb89bc57d88f6a591f013932b3661ea58f9d480).